### PR TITLE
Update 3-2 to follow BASHER

### PR DESCRIPTION
**Pull request by derisen**
## Purpose

* Update project to follow BASHER
* Update all dependencies
* Regenerate the README

## Does this introduce a breaking change?

```
[ x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x ] Feature
[ x ] Code style update (formatting, local variables)
[ x ] Refactoring (no functional changes, no api changes)
[ x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 3-Authorization-II/2-call-api-b2c/README.md [hunk 1]
```diff
@@ -76,112 +101,115 @@ For more information and potential issues, see: [HTTPS in .NET Core](https://doc
...
 Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
 
 > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
 
 1. Open the `SPA\src\app\auth-config.ts` file.
-1. Find the key `clientId` and replace the existing value with the application ID (clientId) of `msal-angular-spa` app copied from the Azure portal.
-1. Find the key `protectedResources.todoListApi.scopes` and replace the existing value with the scope of the web API that you have just exposed during the web API registration steps, for example `https://{tenantName}.onmicrosoft.com/{service_clientId}/demo.read`
-
+1. Find the key `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `msal-angular-spa` app copied from the Azure portal.
+1. Find the key `Enter_the_Tenant_Info_Here` and replace the existing value with your Azure AD tenant domain, ex. `contoso.onmicrosoft.com`.
```
Comment (v-michaelmi) [inline 3-Authorization-II/2-call-api-b2c/README.md:196]: These aren't in the file.

Instead of `Enter_the_...` there's a GUID.

Comment (v-michaelmi) [reply 3-Authorization-II/2-call-api-b2c/README.md:196]: And a reference to the `b2cPolicies` section.

Comment (derisen) [reply 3-Authorization-II/2-call-api-b2c/README.md:196]: Yeah we need to tweak the codegen a little to use a different text in the case of B2C -will update

File: 3-Authorization-II/2-call-api-b2c/API/TodoListAPI.Tests/TodoListAPI.Tests.csproj [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Include="..\TodoListAPI\appsettings.json" Link="appsettings.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
```
Comment (v-michaelmi) [inline 3-Authorization-II/2-call-api-b2c/API/TodoListAPI.Tests/TodoListAPI.Tests.csproj:14]: This shouldn't be necessary?

**Review by salman90 — APPROVED** [review]
LGTM!, same comments as 1-2

**Review by kalyankrishna1 — APPROVED** [review]
:shipit:

File: 3-Authorization-II/2-call-api-b2c/API/TodoListAPI.Tests/TodoListAPI.Tests.csproj (hunk 2 above)
Comment (derisen) [reply 3-Authorization-II/2-call-api-b2c/API/TodoListAPI.Tests/TodoListAPI.Tests.csproj:14]: It is -we are referring to appsettings.json from the main project in the tests

File: 3-Authorization-II/2-call-api-b2c/README.md (hunk 1 above)
Comment (derisen) [reply 3-Authorization-II/2-call-api-b2c/README.md]: done!

**Pull request merged**